source/includes/extracts-migration-to-database-tools.yaml

```diff
@@ -0,0 +1,73 @@
+ref: _migration-to-database-tools
+content: |
+  .. admonition:: Upcoming Tools Migration in MongoDB 4.4
+    :class: note
+
+    Starting in MongoDB 4.4, the ``{{command}}`` documentation 
+    migrates to `MongoDB Database Tools - {{command}} 
+    <https://docs.mongodb.com/database-tools/{{command}}>`__. The
+    latest version of ``{{command}}`` (tool version 100.0.0 and
```

Comment:
@andf-mongodb  -- Oh, the latest could be construed to mean the current latest.  I would use the word "upcoming" since we're already using it in the title -- The upcoming version of ...

Reply:
Removed this line entirely in favor of ...